pagopa/io-app
# chore: [IOBP-490] Removal of BPD folder

## Short description
removal of the whole BPD folder, along with some weird references here and there

DEPENDS ON https://github.com/pagopa/io-app/pull/5421

## List of changes proposed in this pull request
- removed the *whole* folder
- removed feature flags in the env files
- removed references to BPD around the app

the api definitions for the whole bonus section will be removed in a future PR

## How to test
same as last PR, take a stroll around the app and make sure nothing breaks; theoretically *the user* should not notice any difference.
stay on the lookout especially in:
- cgn onboarding
- payment method onboarding
- payment flow
- messages
- idpay flows
